velox/exec/fuzzer/WriterFuzzer.cpp

```diff
@@ -263,9 +294,6 @@ std::vector<RowVectorPtr> WriterFuzzer::generateInputData(
   auto inputType = ROW(std::move(names), std::move(types));
   std::vector<RowVectorPtr> input;
 
-  // For partition keys, limit the distinct value to 4 to avoid exceeding
```

Comment:
This comment still applies?

Reply:
I adjusted kMaxPartitionsPerWritersSession from 100 to 400 to give use room for more bucket (up to 5), just adjusted the comment accordingly.